code/src/index.css

```diff
@@ -1,13 +1,151 @@
+:root {
+  --gray-black: #242626;
+}
+
+#wrapper {
+  width: 90vw;
+  margin: 0 auto;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
+a {
+  text-decoration: none;
+}
+
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.header {
+  color: #ffffff;
+  margin: 0;
+  border-bottom: 2px solid rgba(255, 255, 255, 0.1);
+  margin-bottom: 1em;
+}
+
+.app-section {
+  background-color: var(--gray-black);
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  justify-content: center;
+  grid-column-gap: 2em;
+  grid-row-gap: 2em;
+}
+
+/* ALBUM COVER */
+.cover {
+  position: relative;
+}
+
+.cover-img {
+  width: 100%;
+}
+
+.cover:hover .icons {
+  opacity: 1;
+}
+
+/* ALBUM TITLE */
+.title {
+  color: #ffffff;
+  margin: 0;
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+}
+
+.title:hover {
+  text-decoration: underline 1px;
+}
+
+/* ARTIST NAME */
+.artist-name {
+  color: #a0a0a0;
+}
+
+.artist-name a {
+  color: #a0a0a0;
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+}
+
+.artist-name:hover {
+  text-decoration: underline 1px;
+}
+
+/* ICONS */
+.icons {
+  display: none;
+}
+
+.icon {
+  padding: 0 5%;
+}
+
+.icons {
+  display: flex;
+  justify-content: space-evenly;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  transition: 0.5s ease;
+  opacity: 0;
+  background: rgba(1, 1, 1, 0.64);
+}
+
+.icons-small {
+  width: 11%;
```

Comment:
Is there a specific reason that you used % on the icons? Just referencing what came up during our demons, % could cause problems with Netlify in certain cases, so it may be easier to use px, em or rem for small elements such as icons.